### ISSUE TITLE
Rollbar Client Name Length Restriction

### ISSUE THREAD

**Issue by spencerkittleson** [LambdaSharp/LambdaSharpTool#131]
Project names over 32-35 characters cause the following exception:
```json
    "Message": "Custom::LambdaSharpRegistrationModule: CREATE operation FAILED [One or more errors occurred. (http operation failed: UnprocessableEntity)]",
    "Raw": null,
    "Traces": [
        {
            "Exception": {
                "Type": "System.Exception",
                "Message": "http operation failed: UnprocessableEntity",
                "StackTrace": "   at LambdaSharp.Core.RollbarApi.RollbarClient.CreateProject(String projectName) in C:\\LambdaSharp\\LambdaSharpTool\\Modules\\LambdaSharp.Core\\CommonLib\\RollbarApi\\RollbarClient.cs:line 167\n   at LambdaSharp.Core.RegistrationFunction.Function.ProcessCreateResourceAsync(Request`1 request) in C:\\LambdaSharp\\LambdaSharpTool\\Modules\\LambdaSharp.Core\\RegistrationFunction\\Function.cs:line 109"
            },
            "Frames": [
                {
                    "FileName": "C:\\LambdaSharp\\LambdaSharpTool\\Modules\\LambdaSharp.Core\\CommonLib\\RollbarApi\\RollbarClient.cs",
                    "LineNumber": 167,
                    "ColumnNumber": null,
                    "MethodName": "MoveNext"
                },
                {
                    "FileName": "System.Runtime.ExceptionServices.ExceptionDispatchInfo",
                    "LineNumber": null,
                    "ColumnNumber": null,
                    "MethodName": "Throw"
                },
                {
                    "FileName": "System.Runtime.CompilerServices.TaskAwaiter",
                    "LineNumber": null,
                    "ColumnNumber": null,
                    "MethodName": "HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task task)"
                },
                {
                    "FileName": "C:\\LambdaSharp\\LambdaSharpTool\\Modules\\LambdaSharp.Core\\RegistrationFunction\\Function.cs",
                    "LineNumber": 109,
                    "ColumnNumber": null,
                    "MethodName": "MoveNext"
                }
            ]
        },
        {
            "Exception": {
                "Type": "System.AggregateException",
                "Message": "One or more errors occurred. (http operation failed: UnprocessableEntity)",
                "StackTrace": "   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)\n   at LambdaSharp.CustomResource.ALambdaCustomResourceFunction`2.ProcessMessageStreamAsync(Stream stream) in C:\\LambdaSharp\\LambdaSharpTool\\src\\LambdaSharp\\CustomResource\\ALambdaCustomResourceFunction.cs:line 151"
            },
            "Frames": [
                {
                    "FileName": "System.Threading.Tasks.Task`1[TResult]",
                    "LineNumber": null,
                    "ColumnNumber": null,
                    "MethodName": "GetResultCore(System.Boolean waitCompletionNotification)"
                },
                {
                    "FileName": "C:\\LambdaSharp\\LambdaSharpTool\\src\\LambdaSharp\\CustomResource\\ALambdaCustomResourceFunction.cs",
                    "LineNumber": 151,
                    "ColumnNumber": null,
                    "MethodName": "MoveNext"
                }
            ]
        }
    ]
}
```

**Comment by bjorg** [comment]
Did you reach out to Rollbar support about this. I get it that there should probably be a limit on project name length, but 32-35 characters seems a tad severe!

**Comment by smyleeface** [comment]
I (finally) found it in their docs.
https://explorer.docs.rollbar.com/#operation/create-a-project
```Name of the project. Must start with a letter; can contain letters, numbers, spaces, underscores, hyphens, periods, and commas. Max length 32 characters.```

**Comment by bjorg** [comment]
Well, the fix is going to be annoying, but it should be doable. I'm thinking something like this:
* For project names with 32 or less characters, use the project name as is.
* For project names longer than 32 characters, use the first 26 characters of the project name and a 6 character hash of the full name.

Thoughts?

**Comment by bjorg** [comment]
Fixed in 0.7.0.8.
